src/test/archive/sample_db/dune

```diff
@@ -0,0 +1,14 @@
+(rule
+ (alias archive_db)
+ (deps latest_version)
+ (targets archive_db.sql precomputed_blocks.tar.xz)
+ (action
+  (progn
+   (run bash -c "
+    curl -L -o archive_db.sql \
```

Comment:
Could you silent the output of `curl` using `-q`? As a "sand" task, we're cleaning the build logs, see https://github.com/MinaProtocol/mina/pull/17529.